docs/copied-from-beats/docs/security/users.asciidoc

```diff
@@ -197,156 +344,245 @@ information.
+. If you're using any of the APM, Dashboard, Visualize, Discover, etc., apps,
+you'll need assign space privileges:
 +
 [options="header"]
 |====
 |Type | Privilege | Purpose
 
-ifndef::apm-server[]
-|Cluster
-|`monitor`
-|Retrieve cluster details (e.g. version) 
-endif::apm-server[]
+| Spaces
+| `Read` or `All` on {beat_kib_app}
+| Allow the use of the {beat_kib_app}
 
-ifndef::no_ilm[]
-|Cluster
-|`read_ilm`
-| Read the ILM policy when connecting to clusters that support ILM.
-Not needed when `setup.ilm.check_exists` is `false`.
-endif::no_ilm[]
+| Spaces
+| `Read` or `All` on Dashboards, Visualize, and Discover
+| Allow the user to view, edit, and create dashboards, as well as browse data.
+|====
++
+. Finally, assign the following role if a user needs to enable and edit machine learning features:
++
+[options="header"]
+|====
+|Role | Purpose
 
-ifeval::["{beatname_lc}"=="filebeat"]
-|Cluster
-|`cluster:admin/ingest/pipeline/get`
-|Check for ingest pipelines used by modules. Needed when using modules.
-endif::[]
+|`machine_learning_admin`
+|NEED DESCRIPTION
+|====
 
-|Index
-|`create_doc` on +{beat_default_index_prefix}-*+ indices
-|Write events into {es}
+////
+***********************************  ***********************************
+***********************************  ***********************************
+////
 
-ifndef::no_ilm[]
-|Index
-|`view_index_metadata` on +{beat_default_index_prefix}-*+ indices
-|Check for alias when connecting to clusters that support ILM.
-Not needed when `setup.ilm.check_exists` is `false`.
-endif::no_ilm[]
+[[privileges-agent-central-config]]
+=== Grant privileges and roles needed for APM Agent central configuration
 
-|Index
-|`create_index` on +{beat_default_index_prefix}-*+ indices
-|Create daily indices when connecting to clusters that do not support ILM.
-Not needed when using ILM.
-|====
-ifndef::apm-server[]
-+
-Omit any privileges that aren't relevant in your environment.
-endif::apm-server[]
+++++
+<titleabbrev>Create a _central config_ user</titleabbrev>
+++++
 
-. Assign the *writer role* to users who will index events into {es}.
+[[privileges-agent-central-config-server]]
+==== APM Server central configuration management
 
-[[kibana-user-privileges]]
-==== Grant privileges and roles needed to read {beatname_uc} data
+[options="header"]
+|====
+|Type | Privilege | Purpose
 
-{kib} users typically need to view dashboards and visualizations that contain
-{beatname_uc} data. These users might also need to create and edit dashboards
-and visualizations.
-ifdef::has_central_config[]
-If you're using Beats central management, some of these users might need to
-create and manage configurations.
-endif::has_central_config[]
+| Spaces
+|`Read` on {beat_kib_app}
+|Allow the use of the {beat_kib_app}
+|====
 
-To grant users the required privileges:
+[[privileges-agent-central-config-kib]]
+==== Kibana central configuration management
```

Comment:
Sorry for repeating myself, but this would also be a section worth moving to Kibana in the long run.